spec/supplejack/concept_spec.rb

```diff
@@ -128,5 +128,12 @@ module Supplejack
         end
       end
     end
+
+    describe '#all' do
+      it 'calls the get method' do
```

Comment:
I agree it is old syntax, but the whole file is using it, so I've kept it line with that. 

Also, yes it is just coverage.  I was asked to put at least one test in.